content/locales/en.ts

```diff
@@ -181,11 +188,12 @@ const en = {
     updatePassword: "Update your password",
   },
   "/account/security/update": {
-    title: "Password",
+    title: "Security",
     setBreadcrumb: "Password | set",
     updateBreadcrumb: "Password | update",
     set: "set",
     update: "update",
+    ...passwordInputsStrings,
```

Comment:
We don't need to extract it here.
We needed to do it on profile forms because we have edit and new which uses the same strings.